tests/wolfram_alpha_test.py

```diff
@@ -9,12 +9,24 @@
 
 class WolframAlphaTest(IsolatedTestCase):
 
+    def setUp(self):
+        extra_config = {
+            'DEFAULT_CONFIG': {
+                'WolframAlpha': {
+                    'WA_TOKEN': None,
```

Comment:
why this?

That is the default, so an override shouldnt be necessary, iiuc

Reply:
Yes, this was excessive, removed it.